itest/lnd_mpp_test.go

```diff
@@ -176,8 +178,8 @@ type mppTestScenario struct {
 //
 //	            _ Eve _
 //	           /       \
-//	Alice -- Carol ---- Bob
-//	    \              /
+//	Alice -- Carol ---- Bob ---------- Paula
+//	    \              /     (private)
```

Comment:
it would be great if you could put these preparatory changes into its own commit

Reply:
done